eng/pipelines/docs-to-blob.yml

```diff
@@ -0,0 +1,40 @@
+#variables => $(TargetPipelineProjectId), $(TargetPipelineDefinition), $(TargetPipelineBranch), $(azure-sdk-doc-sas-key) (from keyvault)
```

Comment:
Where are these variables getting set?

Reply:
In the UI on Devops

Reply:
Actually we are deleting this and instead using a common script for all languages under `azure-sdk-build-tools`